Jenkinsfile

```diff
@@ -180,6 +181,14 @@ def assertAppIsDeployed(appName) {
   }
 }
 
+def assertBranding() {
+  waitUntilAppIsReady('demo-portal')
+  def customCss = sh (script: "wget -qO- http://localhost:8080/demo-portal/faces/javax.faces.resource/custom.css?ln=xpertivy-branding", returnStdout: true)
```

Comment:
we should rename this someday "xpertivy"
fyi @ivy-cst

Reply:
A little bit history in our code ;)